configuration/scripts/ice_in

```diff
@@ -171,11 +171,11 @@
     fe_data_type    = 'default'
     fyear_init      = 1997
     ycycle          = 1
-    atm_data_dir    = '/glade/u/home/tcraig/cice_data/'
-    ocn_data_dir    = '/unknown_ocn_data_dir'
-    bgc_data_dir    = '/uknown_bgc_data_dir'
     atm_data_format = 'bin'
+    atm_data_dir    = '/glade/u/home/tcraig/cice_data/'
```

Comment:
Should this really point to your directory, @apcraig ?
ICE_MACHINE_INPUTDATA/CICE_data/ might be better.

Reply:
I noticed that too.  It's been that way for a while.  I'll fix it sometime.  It's always overridden so I don't think it's a problem.  But I agree better to have something more generic there.